frontend/ui/widget/inputdialog.lua

```diff
@@ -145,6 +166,8 @@ function InputDialog:init()
             align = "left",
             self.title,
             self.title_bar,
+            -- description,
```

Comment:
nitpick, in case you are going to do another round of code change, otherwise i will change it myself in my next PR.

this comment line is redundant.